src/test_rand_types.rs

```diff
@@ -0,0 +1,324 @@
+#![cfg(test)]
+
+use hashes::Hash;
+use network::message_network::{VersionMessage, Reject, RejectReason};
+use network::Address;
+use network::message_blockdata::{InvType, Inventory, GetBlocksMessage, GetHeadersMessage};
+use network::constants::{ServiceFlags};
+use secp256k1::rand::distributions::{Distribution, Standard, Alphanumeric};
+use secp256k1::rand::Rng;
+use secp256k1::rand::seq::SliceRandom;
+use std::iter;
+use {Transaction, TxIn, OutPoint, Script};
+use ::{TxOut, Block};
+use BlockHeader;
+use network::message_filter::{GetCFilters, CFilter, GetCFHeaders, CFHeaders, GetCFCheckpt, CFCheckpt};
+use network::message::{CommandString, NetworkMessage};
```

Comment:
I think the most common way to structure imports (in rust-bitcoin and perhaps in Rust) is in 3 blocks: std/core imports, external deps imports, internal imports. Maybe my observation regarding its popularity is biased because I like it that way, but I've definitely seen it elsewhere as well :)